lib/peep.rb

```diff
@@ -0,0 +1,33 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :message, :time
+
+  def initialize(message:, time:)
+    @message = message
+    @time = time
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    result = connection.exec("SELECT * FROM peeps ORDER BY time DESC;")
+    result.map do |peep|
+      Peep.new(message: peep['message'], time: peep['time'])
+    end
+  end
+
+  def self.write(text:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
It would be really nice if you could store your db name as an env variable, so you could do something along the lines of:
```ruby
db_name = ENV['DB_NAME']
connection = PG.connect(dbname: db_name)
```